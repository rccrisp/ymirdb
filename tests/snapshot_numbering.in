set a 1 2 3
snapshot
set b 4 5 6
snapshot
set c 7 8 9
snapshot
list snapshots
drop 3
list snapshots
snapshot
list snapshots
drop 5
bye
