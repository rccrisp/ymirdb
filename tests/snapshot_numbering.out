> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> saved as snapshot 3

> 3
2
1

> ok

> 2
1

> saved as snapshot 4

> 4
2
1

> no such snapshot

> bye
